src/service/dialogflow/dialogflow.ts

```diff
@@ -439,6 +439,9 @@ export const dialogflow: Dialogflow = <
     while (typeof handler !== 'function') {
       if (typeof handler === 'undefined') {
         if (!this._handlers.fallback) {
+          if (!intent) {
+            throw new Error('No intent was provided and fallback intent is not defined.')
```

Comment:
Can you change `fallback intent` to `fallback handler`? That's what's being checked here.

Reply:
Good point, thanks!